cime_config/testdefs/testlist_cam.xml

```diff
@@ -26,24 +26,18 @@
       <option name="wallclock">00:10:00</option>
     </options>
   </test>
-  <test compset="NFLTHISToslo_v0d" grid="ne3pg3_ne3pg3_mg37" name="PEM_D_Ln9_P18" testmods="cam/outfrq9s">
-    <machines>
-      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
-      <machine name="betzy" compiler="intel" category="cam_bfb"/>
-    </machines>
-    <options>
-      <option name="wallclock">00:10:00</option>
-    </options>
-  </test>
 
-  <test compset="NF2000dev" grid="ne3pg3_ne3pg3_mg37" name="SMS_D_Ln9_P36" testmods="cam/outfrq9s">
-    <machines>
-      <machine name="betzy" compiler="intel" category="aux_cam_noresm"/>
-    </machines>
-    <options>
-      <option name="wallclock">00:10:00</option>
-    </options>
-  </test>
+  <!-- NOTE: the following commented out test assumes a 2000_cam6.xml which does not work with camdev - so another use -->
+  <!-- case is needed -->
+  <!-- <test compset="NF2000" grid="ne3pg3_ne3pg3_mg37" name="SMS_D_Ln9_P36" testmods="cam/outfrq9s"> -->
+  <!--   <machines> -->
+  <!--     <machine name="betzy" compiler="intel" category="aux_cam_noresm"/> -->
+  <!--   </machines> -->
+  <!--   <options> -->
+  <!--     <option name="wallclock">00:10:00</option> -->
+  <!--   </options> -->
+  <!-- </test> -->
+
```

Comment:
I'm not sure this is ever coming back, okay to just delete test?

Reply:
done.